MaterialLib/SolidModels/LinearElasticIsotropic.h

```diff
@@ -0,0 +1,104 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef MATERIALLIB_SOLIDMODELS_LINEARELASTICISOTROPIC_H_
+#define MATERIALLIB_SOLIDMODELS_LINEARELASTICISOTROPIC_H_
+
+#include "SolidsBase.h"
+
+namespace Solids
+{
+template <int DisplacementDim>
+class LinearElasticIsotropic final : public SolidsBase<DisplacementDim>
+{
+public:
+    /// Variables specific to the material model
+    class MaterialProperties
```

Comment:
OK. The suggestion was to make a **not** nested class LinearElasticIsotropicMaterialProperties (or something with shorter name).